pkg/controllers/logs/controller.go

```diff
@@ -36,9 +39,38 @@ func NewLogsController(log *zap.Logger, logsProvider logs.LogsProvider, router *
 	r.GET("", controller.Logs)
 	r.GET("/logs/namespace/:namespace/:entity/:entity_name", controller.FilterEntityLogs)
 	r.GET("/logs_by_labels/:labels", controller.FilterLabelLogs)
+	///r1 := router.Group("")
```

Comment:
Please remove if this code is unused

Reply:
sorry for this glitch. Removed the unused and redundant parts.